src/frontend/js/widgets/Dashboard/components/TeacherDashboardCourseSidebar/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { TeacherDashboardPaths } from 'widgets/Dashboard/utils/teacherRouteMessages';
+
+interface GetMenuRoutesArgs {
+  courseProductRelationId?: string;
+  organizationId?: string;
+}
+export const getMenuRoutes = ({ courseProductRelationId, organizationId }: GetMenuRoutesArgs) => {
+  if (organizationId) {
+    if (courseProductRelationId) {
+      return [
+        TeacherDashboardPaths.ORGANIZATION_PRODUCT,
+        TeacherDashboardPaths.ORGANIZATION_PRODUCT_CONTRACTS,
+      ];
+    }
+    return [
+      TeacherDashboardPaths.ORGANIZATION_COURSE_GENERAL_INFORMATION,
+      TeacherDashboardPaths.ORGANIZATION_COURSE_CONTRACTS,
+    ];
```

Comment:
```suggestion
return courseProductRelationId
      ? [
          TeacherDashboardPaths.ORGANIZATION_PRODUCT,
          TeacherDashboardPaths.ORGANIZATION_PRODUCT_CONTRACTS,
        ]
      : [
          TeacherDashboardPaths.ORGANIZATION_COURSE_GENERAL_INFORMATION,
          TeacherDashboardPaths.ORGANIZATION_COURSE_CONTRACTS,
        ];
```

Reply:
i think the early return is easier to read.